koku/masu/external/downloader/azure/azure_report_downloader.py

```diff
@@ -213,11 +216,13 @@ def _get_manifest(self, date_time):  # noqa: C901
                 last_modified = blob.last_modified
                 LOG.info(log_json(self.tracing_id, f"Found cost export {report_name}", self.context))
                 manifest["reportKeys"] = [report_name]
-        try:
-            manifest["assemblyId"] = extract_uuids_from_string(report_name).pop()
-        except IndexError:
-            message = f"Unable to extract assemblyID from {report_name}"
-            raise AzureReportDownloaderError(message)
+
+        if not manifest["assemblyId"]:
```

Comment:
```suggestion
```
Since we appear to only populate the assembly_id under `if self.ingress_reports`, let's just move this try/except to under the `else`. (Which is where it is if you just delete this `if` statement.